src/helpers/create-wallets-for-provider.ts

```diff
@@ -32,15 +32,12 @@ export async function createWalletsForProvider(
       canNext = balance.isPositive() || index === 0;
 
       if (canNext) {
-        await Wallet.create(
-          {
-            address: address,
-            type: walletType,
-            path: hdPath,
-            accountId: provider.getIdentifier(),
-          },
-          name,
-        );
+        Wallet.create(name, {
```

Comment:
Create is an async function
```suggestion
        await Wallet.create(name, {
```